src/controllers/check.your.answers.controller.ts

```diff
@@ -18,9 +20,21 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
     const appData: ApplicationData = getApplicationData(req.session);
 
+    let hasTrusts: boolean = false;
+
+    if (isActiveFeature(config.FEATURE_FLAG_ENABLE_TRUST_INFO_16062022)) {
+      hasTrusts = checkEntityHasTrusts(appData);
+    }
+
```

Comment:
I would also log out whether any trusts are present or not (line 28) 
something like logger.infoRequest(req, `config.CHECK_YOUR_ANSWERS_PAGE hasTrusts=${hasTrusts}`);